ocs_ci/ocs/monitoring.py

```diff
@@ -171,3 +171,36 @@ def check_ceph_health_status_metrics_on_prometheus(mgr_pod):
         [mgr_pod for health_status in ceph_health_metric.get('data').get(
             'result') if mgr_pod == health_status.get('metric').get('pod')]
     )
+
+
+def prometheus_health_check(name='monitoring', kind='ClusterOperator'):
```

Comment:
Please same 'monitoring' and 'ClusterOperator' as constants

Reply:
defined in constants and used